extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -346,7 +346,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       const recipient = orderedRecipients[processed];
       const escapedTitle = Xss.escape(recipient.element.getAttribute('title') || '');
       const nameOrEmail = recipient.name || recipient.email || recipient.invalid || '';
-      const emailHtml = `<span class="email_address ${recipient.element.className}" title="${escapedTitle}">${Xss.escape(nameOrEmail)}</span>`;
+      const emailHtml = `<span class="email_address ${recipient.element.className}" title="${escapedTitle}">${Xss.stripEmojis(nameOrEmail)}</span>`;
```

Comment:
in initial review I meant that we don't need too use `stripEmojis` each time when recipient is rendered in UI. as issue happens only when sending message to recipient with emoji using FES.

so in UI we can leave emojis and strip them only when formatting password-protected message for FES. it'll be simpler, but will fix original issue

Reply:
Oh, okay - I just thought it would be great to have a consistent UI but yes, I'm reverting that change. thank you.